static/src/js/util/map/interactions/onClickShapefile.ts

```diff
@@ -112,14 +112,18 @@ const onClickShapefile = ({
     // Add the feature to the selectedFeatures
     selectedFeatures = updatedSelectedFeatures.concat(edscId)
   } else {
-    // Remove the feature from the existing spatial query
-    const queryByType = [...currentQuery[queryType] || []]
-    const queryIndex = queryByType.indexOf(queryValue)
-    queryByType.splice(queryIndex!, 1)
+    // Remove all queryValues from the existing spatial query (for multi-geometries)
+    const currentQueryValue = [...currentQuery[queryType]]
+    queryValues.forEach((queryValue) => {
```

Comment:
We should always have blank lines between variable declarations and blocks. Same issue exists with lines 118 and 119